go.mod

```diff
@@ -8,137 +8,198 @@ go 1.12
 
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
-	github.com/Azure/azure-sdk-for-go v21.4.0+incompatible
-	github.com/Azure/go-autorest v11.1.2+incompatible
-	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20181220005116-f8e995905100
+	cloud.google.com/go v0.45.1 // indirect
```

Comment:
Do we understand what is causing all of these changes?  They seem wrong to me, but I am not as well versed in go modules...

@liggitt

Reply:
Agree. Seems like a bad merge or uncontrolled update of a lot of dependencies. Would recommend squashing, undoing all vendor and go.{mod,sum} changes, then repinning k8s.io/utils and running update-vendor.sh

Reply:
Yep - I think I did some changes a while ago, forgot about them, and it got included here (learning so newbie mistake).. I'll try and do what you suggested.